prometheus/summary.go

```diff
@@ -112,7 +112,9 @@ type SummaryOpts struct {
 	ConstLabels Labels
 
 	// Objectives defines the quantile rank estimates with their respective
-	// absolute error. The default value is DefObjectives.
+	// absolute error. If Objectives[q] = e, then the reported q-quantile
+	// value will be the r-quantile value with r between q-e and q+e.
+	// The default value is DefObjectives.
```

Comment:
The naming of quantiles is very confusing in the literature. An _r-quantile_ would suggest something like '4-quantile', which would refer to quartiles. The most fitting name for the fractional quantile I could find is _φ-quantile_. My co-developers lack the humanist education I have enjoyed and regularly chastise me for using Greek letters, but in this case we concluded it is least confusing after all. Also to be consistent with the documentation on prometheus.io, I'd suggest the following:

```
If Objectives[q] = e, then the value reported for q
will be the φ-quantile value with φ between q-e and q+e.
The default value is DefObjectives.
```